app/Http/Transformers/PostTransformer.php

```diff
@@ -42,10 +42,10 @@ public function transform(Post $post)
                 'text' => $post->text,
             ],
             'quantity' => $post->quantity,
-            'reactions' => [
-                'reacted' => ! empty($post->reaction),
-                'total' => $post->reactions_count,
-            ],
+            // 'reactions' => [
```

Comment:
So we don't end up with errors elsewhere when we test this, let's just return `false` and `0` for each of these.